source/settings/settings-reference.txt

```diff
@@ -0,0 +1,122 @@
+.. _compass-interface-settings:
+.. _compass-settings-reference:
+
+--------------------------
+Compass Interface Settings
+--------------------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can customize |compass| to meet your specific needs on the |compass-short| 
+interface. Some options are only available in the :guilabel:`Settings` panel.
+
+If you have already configured |compass-short| settings on the command line or 
+in a configuration file, you cannot modify those settings in the |compass-short| 
+interface.
+
+Settings Panel Location
+-----------------------
+
+To open the |compass-short| :guilabel:`Settings` panel, navigate to the 
+:guilabel:`MongoDB Compass` system menu and select :guilabel:`Settings`.
+
+.. figure:: /images/compass/settings-panel-location.png
+  :alt: Settings panel location under the MongoDB Compass system menu
```

Comment:
[suggestion]

I'm not sure we need this screenshot to be so large. Consider using either the `scale` or `figwidth` option to scale this down a smidge.

Reply:
Oh didn't know about these options! Thanks for the suggestion 👍